weldx/tests/transformations/test_cs_manager.py

```diff
@@ -1824,6 +1824,7 @@ def test_remove_subsystems(list_of_csm_and_lcs_instances, nested):
     ],
 )
 @pytest.mark.slow
+@pytest.mark.filterwarnings("ignore:The following coordinate systems.*:UserWarning")
```

Comment:
fyi @vhirtham I have disable the specific CSM warning for this test to clean up the output

Reply:
Noticed recently, that there are new warnings, just didn't realize so far where they were introduced since we at all of them gone at some point.